packages/react/src/components/Toolbar/Toolbar.tsx

```diff
@@ -184,6 +185,15 @@ class Toolbar extends UIComponent<WithAsProp<ToolbarProps>> {
       return
     }
 
+    if (
+      this.context.target.activeElement === el ||
+      el.contains(this.context.target.activeElement)
+    ) {
+      if (this.containerRef.current) {
+        this.containerRef.current.querySelector('[tabindex]')
```

Comment:
could you pls check if `getFirstFocusable` from packages\react-bindings\src\FocusZone\focusUtilities.ts would work?

Reply:
What do you think it's better?
```js
        const firstFocusableItem = getFirstFocusable(
          this.containerRef.current,
          this.containerRef.current.firstElementChild,
        )
```
or
```js
        const firstFocusableItem = getFirstFocusable(
          this.containerRef.current,
          this.containerRef.current,
        )
````
Both seem to work in this case.